<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContentsClass</class>
 <widget class="QWidget" name="ContentsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ContentsClass</string>
  </property>
  <widget class="QTextBrowser" name="contentsTextBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>191</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>330</y>
     <width>111</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>저장</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>나가기</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="dateTextBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>256</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ContentsClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>715</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>contentsTextBrowser</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
